frontend/src/components/DsfrSegmentedControl.vue

```diff
@@ -0,0 +1,134 @@
+<template>
+  <fieldset :class="{ 'fr-segmented': true, 'fr-segmented--no-legend': noLegend }">
+    <legend class="fr-segmented__legend">{{ legend }}</legend>
+    <div class="fr-segmented__elements">
+      <div v-for="item in itemsForDisplay" :key="item.value" class="fr-segmented__element">
+        <!-- if this component is used in multiple places on one page, the single name will be a problem -->
```

Comment:
Maybe `name` could be a property?

Reply:
https://github.com/betagouv/ma-cantine/pull/3876/commits/8e4288d7f82efdec68719a2529a8223a072e8cb0